common/compositor/gl/glprogram.cpp

```diff
@@ -128,6 +128,60 @@ static std::string GenerateFragmentShader(int layer_count) {
 
 static GLint GenerateProgram(unsigned num_textures,
                              std::ostringstream *shader_log) {
+  GLint status;
+  GLint program = glCreateProgram();
+  if (!program) {
+    if (shader_log)
+      *shader_log << "Failed to create program."
+                  << "\n";
+    return 0;
+  }
+
+  // try pre-built shader program first
+  std::ostringstream shader_program_fname;
+  shader_program_fname << "hwc_shader_prog_"
+                       << num_textures
+                       <<".shader_test.bin";
+
+  FILE *shader_prog_fp;
+  static PFNGLPROGRAMBINARYOESPROC program_binary_oes =
+     (PFNGLPROGRAMBINARYOESPROC)eglGetProcAddress("glProgramBinaryOES");
```

Comment:
For consistency, can you place this with the other get_proc calls in common/compositor/gl/shim.cpp

Reply:
ok